tests/prison-custody-status-to-delius/release-recall-test.spec.ts

```diff
@@ -57,7 +57,7 @@ test('Release and recall test', async ({ page }) => {
 
 test('Temporary absence test', async ({ page }) => {
     test.slow() // increase the timeout - releases/recall publishing can take a few minutes
-
+    page.on('console', msg => console.log(msg.text()))
```

Comment:
You'll need to remove this, otherwise we'll still log the auth token to the console